### ISSUE TITLE
[APIM 3.2] Missing steps when enabling usersignup workflow for tenants

### ISSUE THREAD

**Issue by himasha** [wso2/docs-apim#2130]
**Description:**
Following guide[1] should be updated with the extra steps required if there are multiple tenants and you need to enable this workflow for each tenant. 

1. At tenant admin's login in carbon console  https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup/#enabling-self-signup should be configured.
2. At tenant admin's login in carbon console following file (/_system/governance/apimgt/applicationdata/workflow-extensions.xml) should be updated with the new workflow executor. 

Basically need a note stating that in a multi tenant environment, following steps should be followed.
**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by Mariangela** [comment]
@himasha , what are you referring to as [1]?? I do not see a link

**Comment by himasha** [comment]
@Mariangela 
[1] https://apim.docs.wso2.com/en/next/develop/customizations/adding-a-user-signup-workflow/

**Comment by pubudu538** [comment]
@Jayanie is working on this issue.
